osbs/build/build_request.py

```diff
@@ -867,7 +889,13 @@ def render(self, validate=True):
                            # add the path to the plugin's
                            # configuration. This is done elsewhere.
                            None):
-                          self.spec.registry_secrets.value})
+                          self.spec.registry_secrets.value,
+
+                          ('buildstep_plugins', 'orchestrate_build', 'osbs_client_config'):
```

Comment:
@lcarva, let's not mount over /etc. Don't forget, we also use dockpulp.conf when Pulp integration is enabled, as well as /etc/yum.repos.d/ which we'd have to recreate.